index.md

```diff
@@ -302,37 +302,50 @@ host myfancyrfp {
 Example snippet for `dnsmasq-dhcp-server`'s `dnsmasq.conf`:
 
 ```
-#Not strictly need just makes dnsmasq ignore any wishes the rfp has.
+# set interface dnsmasq listens to
+interface=enp0s31f6
+
+# set dnsmask as the authoritative dhcp server
 dhcp-authoritative
 
-#The RFP host definiton, MAC, IP, HOSTNAME
-dhcp-host=00:30:42:cc:23:42,10.10.10.10,myfancyrfp
+# dhcp pool (start, end, netmask, leasetime)
+dhcp-range=10.10.10.100,10.10.10.199,255.255.255.0,12h
 
-#The location of the leasefile
-dhcp-leasefile=/var/lib/dnsmasq/dnsmasq.leases
+# static leases for RFPs
+# for every RFP, set a tag to choose on which tftp boot file to take (tags are definded below: rfp2g/rfp3g/rfp4g)
+dhcp-host=00:30:42:cc:23:42,set:rfp3g,10.10.10.10,rfp-1
+dhcp-host=00:30:42:cc:23:43,set:rfp2g,10.10.10.11,rfp-2
+dhcp-host=00:30:42:cc:23:44,set:rfp2g,10.10.10.12,rfp-3
 
-#If a devices says in the dhcp discover packet, that it is from vendor OpenMobility* the dhcp server will set vendor specific options, which equals the code 43 from the dhcpd example, with the option code 10, and the IP adress. 
-#With out that the omm will just ignore the dhcp offer and send another discover packet. 
-#The 10 says hey you are running the omm now and the ip points the rfp to the omm. 
-#If you want to run the om on an other device you have to change the 10 with a 19, just like in the other example above.
+# tell the RFPs where to find the omm by setting vendor-specific option 10 to rfp-1's adress.
+# this also tells rfp-1 to host the omm on the rfp.
 dhcp-option=vendor:OpenMobility,10,10.10.10.10
 
-#Defines the range of IP-Adresses handed out by dnsmasq with the subnet mask and lease time
-dhcp-range=10.10.10.100,10.10.10.199,255.255.255.0,12h
+# 2nd gen RFPs will only take the dhcp offer when option 224 is set to the magic_str "OpenMobilitySIP-DECT".
+# It might be possible to have co-existing DHCP servers where only the one responsible for RFPs sends the magic_str.
+dhcp-option=224,"OpenMobilitySIP-DECT"
 
-#Interface to listen on
-interface=enp0s31f6
+# set the tftp server adress (option 150) and tftp boot file name (option 67) depending on the tag set in the dhcp-host above
+dhcp-option=150,10.10.10.1
+dhcp-option=tag:rfp2g,67,"iprfp2G.tftp"
+dhcp-option=tag:rfp3g,67,"iprfp3G.dnld"
+dhcp-option=tag:rfp4g,67,"iprfp4G.dnld"
+
+# location of the leasefile
+dhcp-leasefile=/var/lib/dnsmasq/dnsmasq.leases
 
-#Disables dnsmasq on dns server
+# disable dnsmasq's dns server
 port=0
 
-#DNS Server to use
+# DNS servers to use
 server=9.9.9.9
 server=149.112.112.112
-```
 
+# enable built-in tftp server (e.g. for gen 2 boot files, ...)
+enable-tftp
+tftp-root=/srv/tftp
+```
 
-The RFP will ignore DHCP offers that do not contain the `magic_str`. This means that (1) the magic string must be included in DHCP replies intended for RFPs, so (2) it may be possible to have two co-existing DHCP servers (one for normal devices and one for RFPs).
```

Comment:
@vschlegel Was it intentional that you removed this part?
(I see that it's mentioned in a comment in the config section, but people might miss it if it's not in the "plaintext" part of the how-to)

Reply:
I would assume someone using dhcpd won't look at the dnsmasq config and the other way around. But we could put it into it's own section and reference it in the config to save on duplicate effort maintaining.